### PR TITLE
Add a .nvmrc file

### PR DESCRIPTION
**Pull request by mikachan**
I often see Node errors related to using the wrong version when I come back to this project, so here is a small PR to add an .nvmrc file to make it easy to switch to the correct version of Node.

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+18
```
Comment (matiasbenedetto) [inline .nvmrc:1]: Could we match the Gutenberg node required version? https://github.com/WordPress/gutenberg/blob/trunk/.nvmrc
```suggestion
20
```

**Review by matiasbenedetto — COMMENTED** [review]
It would be nice to match the Gutenberg node version and update the [README.md](https://github.com/WordPress/google-fonts-to-wordpress-collection/blob/trunk/README.md) file accordingly.

**Comment by mikachan** [comment]
> It would be nice to match the Gutenberg node version and update the [README.md](https://github.com/WordPress/google-fonts-to-wordpress-collection/blob/trunk/README.md?rgh-link-date=2024-06-21T18%3A09%3A03Z) file accordingly.

Yeah I agree, but I think a few more things need to be updated to use Node 20. Will look into it next and update this PR, thanks!

**Review by matiasbenedetto — APPROVED** [review]
> Yeah I agree, but I think a few more things need to be updated to use Node 20. Will look into it next and update this PR, thanks!

Ok, go ahead  :+1:

**Comment by mikachan** [comment]
It looks like the [woff2 package](https://www.npmjs.com/package/woff2) is dependent on an older version of Node (it works with 18), as I see errors if I try to use Node 20+ to build the fonts:

image

**Comment by vcanales** [comment]
> It looks like the [woff2 package](https://www.npmjs.com/package/woff2) is dependent on an older version of Node (it works with 18), as I see errors if I try to use Node 20+ to build the fonts:

I can reproduce this with node@20. [woff2 package](https://www.npmjs.com/package/woff2) doesn't seem to get a lot of attention dev-wise; would it be worth it to look at alternatives? It's a wrapper of Google's woff2 utility, and there are a couple of other wrappers available on the npm registry:

- https://www.npmjs.com/package/ttf2woff2
- https://www.npmjs.com/package/@chinese-fonts/wawoff2 (WebAssembly build)

**Comment by mikachan** [comment]
Yeah we could consider using an alternative so we're able to use a more recent version of Node. After a quick glance based on weekly downloads, [npmjs.com/package/ttf2woff2](https://www.npmjs.com/package/ttf2woff2) would be my suggestion.

I'm happy to try that out on this PR.

**Comment by mikachan** [comment]
It looks like both [ttf2woff2](https://www.npmjs.com/package/ttf2woff2) and [@chinese-fonts/wawoff2](https://www.npmjs.com/package/@chinese-fonts/wawoff2) only support converting TTF files to WOFF2, whereas we need to convert WOFF2 to TTF.

I also looked at [woff2-next](https://www.npmjs.com/package/woff2-next), which seems to require an even older version of Node.

I'm struggling to find an alternative package that converts WOFF2 to TTF. Should we just add an .nvmrc file for now, to make the devex slightly better for this repo?

**Review by vcanales — APPROVED** [review]
> Should we just add an .nvmrc file for now, to make the devex slightly better for this repo?

Yeah, sounds to me like we're stuck with `node@18` until we find a replacement for the `woff2` package.

**Comment by creativecoder** [comment]
>  whereas we need to convert WOFF2 to TTF.

This is only for the purpose of generating svg previews, right? In that case, maybe we should just download the ttf version of the font from google and use that to generate previews (in a separate PR).

**Comment by mikachan** [comment]
> This is only for the purpose of generating svg previews, right? In that case, maybe we should just download the ttf version of the font from google and use that to generate previews (in a separate PR).

Yeah, that's all it's for. This sounds like a good plan. 

I'll go ahead and merge this for now, thanks all!

**Comment by creativecoder** [comment]
Issue for deprecating the `woff` npm package here: https://github.com/WordPress/google-fonts-to-wordpress-collection/issues/33

**Pull request merged**